res/entities/Player.json

```diff
@@ -4,13 +4,13 @@
     {
       "componentType": "Player"
     },
-	{
+	  { 
       "componentType": "Sprite",
       "src": "entity/player_modelDefault",
-	  "origin": {
-		"x": 18,
-		"y": 48
-	  },
+	    "origin": {
+		    "x": 15,
```

Comment:
What IDE/editor are you using? Install some kind of beautify/autoindent plugin and you're set.